src/assets/data/categories.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "id": 0,
```

Comment:
Remove starters, drinks and soups as they don't belong in categories (add things like soft drinks, wines, seafood, fish, ect...)

Reply:
Resolved